zkevm-circuits/src/evm_circuit/param.rs

```diff
@@ -1,6 +1,6 @@
 // Step dimension
 pub(crate) const STEP_WIDTH: usize = 32;
-pub(crate) const STEP_HEIGHT: usize = 10;
+pub(crate) const STEP_HEIGHT: usize = 14;
```

Comment:
The increase in height is due to `BeginTx` circuit?

Reply:
Yes

Reply:
Then couldn't we have `STEP_HEIGHT` as the sum of the constants that build it? On that case is easier to track this kind of modifications.

Reply:
The desired cells are `STEP_WIDTH * STEP_HEIGHT`, and we could tune the dimension to optimize the verifier cost.

For example (assume extra rotation is almost free for verifier thanks for SHPLONK), if we find out the extra column is required to have less rotation to verify more steps, then we can have a wider circuit. In the contrary, if we find we don't need so many steps, we can have a narrower circuit.

So I think it's more like a parameter than a value inferred from the gadget, as for why width is `32`, it's a naively value to have at least 1 million steps.

Reply:
Hmmm I see. 
Seems that it's too complex then to compute these constants adding the contributions of the gadget compositions they represent.

I said it as it would be useful and much easier if `14` doesn't appear due to "magic" but instead appears from the computation: 
```rust
pub(crate) const STEP_HEIGHT: usize = BeginTxGadget::HEIGHT_CONTRIBUTION + OtherGadget::HEIGHT_CONTRIBUTION;
```

This was what I meant. But I can be missing some stuff so no worries.

Reply:
I see, but the gadgets actually care how many cells they need than the height. Will think about how to manage this parameter more properly

Reply:
But they're known to use some `offset` parameter which will determine the height no? Or at least a max_height that they can go up to.

Anyway, it's not needed for this PR. Just wanted to give the idea at least. But we can address it later! :)